components/serial-libs/netcdf-cxx/SPECS/netcdf-cxx4.spec

```diff
@@ -0,0 +1,163 @@
+
+# Base package name
+
+%define pname netcdf-cxx
+
+Name:           %{pname}-%{compiler_family}%{PROJ_DELIM}
+Summary:        C++ Libraries for the Unidata network Common Data Form
+License:        NetCDF
+Group:          %{PROJ_NAME}/io-libs
+Version:        4.3.1
+Release:        1%{?dist}
+Url:            http://www.unidata.ucar.edu/software/netcdf/
+Source0:	https://github.com/Unidata/netcdf-cxx4/archive/v%{version}.tar.gz
+
+BuildRequires:  zlib-devel >= 1.2.5
+Requires:       lmod%{PROJ_DELIM} >= 7.6.1
+BuildRequires:  hdf5-%{compiler_family}%{PROJ_DELIM}
+BuildRequires:  netcdf-%{compiler_family}%{PROJ_DELIM}
+Requires:       netcdf-%{compiler_family}%{PROJ_DELIM}
+
+
+#!BuildIgnore: post-build-checks rpmlint-Factory
+
+# Default library install path
+%define install_path %{OHPC_LIBS}/%{compiler_family}/%{pname}/%version
+
+%description
+NetCDF (network Common Data Form) is an interface for array-oriented
+data access and a freely-distributed collection of software libraries
+for C, Fortran, C++, and perl that provides an implementation of the
+interface.  The NetCDF library also defines a machine-independent
+format for representing scientific data.  Together, the interface,
+library, and format support the creation, access, and sharing of
+scientific data. The NetCDF software was developed at the Unidata
+Program Center in Boulder, Colorado.
+
+NetCDF data is:
+
+   o Self-Describing: A NetCDF file includes information about the
+     data it contains.
+
+   o Network-transparent:  A NetCDF file is represented in a form that
+     can be accessed by computers with different ways of storing
+     integers, characters, and floating-point numbers.
+
+   o Direct-access:  A small subset of a large dataset may be accessed
+     efficiently, without first reading through all the preceding
+     data.
+
+   o Appendable:  Data can be appended to a NetCDF dataset along one
+     dimension without copying the dataset or redefining its
+     structure. The structure of a NetCDF dataset can be changed,
+     though this sometimes causes the dataset to be copied.
+
+   o Shareable:  One writer and multiple readers may simultaneously
+     access the same NetCDF file.
+
+
+%prep
+%setup -q -n %{pname}4-%{version}
+
+%build
+# OpenHPC compiler/mpi designation
+%ohpc_setup_compiler
+
+module load hdf5
+module load netcdf
+
+export CPPFLAGS="-I$HDF5_INC -I$NETCDF_INC"
+export LDFLAGS="-L$HDF5_LIB -L$NETCDF_LIB"
+
+./configure --prefix=%{install_path} \
+    --enable-shared \
+    --enable-netcdf-4 \
+    --enable-dap \
+    --enable-ncgen4 \
+    --with-pic \
+    --disable-doxygen \
+    --disable-filter-testing \
+    --disable-static || { cat config.log && exit 1; }
+
+make %{?_smp_mflags}
+
+%install
+# OpenHPC compiler/mpi designation
+%ohpc_setup_compiler
+
+module load hdf5
+module load netcdf
+
+make %{?_smp_mflags} DESTDIR=$RPM_BUILD_ROOT install
+
+# Remove static libraries
+#find "%buildroot" -type f -name "*.la" | xargs rm -f
```

Comment:
Should this commented out code be here ?

Reply:
Probably not, but it was in the original.  I'll remove it here.